lib/pdc/v1/release_variant.rb

```diff
@@ -1,13 +1,18 @@
 module PDC::V1
   class ReleaseVariant < PDC::Base
-    self.primary_key = :id
```

Comment:
I tried to do it in the first place, actually it may be (:release)/(:uid). It could work well for generate the resource url, `release` and `:uid` will be replaced the property value of `release` and `uid`.  But not good in other scenario, like `where`.

I think the expand work like this, if it find the keyword value in attributes, it will replace the uri's keyword with this value. If this keyword not found in attributes, it will generate a url like `?this_keyword=this_value` url. So if we set uri like this, `ReleaseVariant.where(release: id)` will be expanded to `release-variants/id` instead of `release-variants/?release=id`.

So it looks like we still need to hack in somewhere, maybe `where`? I am not sure if we need to do it. Or maybe there is a simple way to do it I can't find yet.